boards/arm/96b_argonkey/doc/96b_argonkey.rst

```diff
@@ -0,0 +1,164 @@
+.. _96b_argonkey:
+
+96Boards Argonkey
+################
+
+Overview
+********
+
+96Boards Argonkey board is based on the STMicroelectronics STM32F412CG
+Cortex M4 CPU. Zephyr applications use the 96b_argonkey configuration
+to run on these boards.
+
+This board acts as a sensor hub platform for all 96Boards compliant
+family products. It can also be used as a standalone board.
+
+.. figure:: img/96b-argonkey-front.jpg
+     :width: 557px
+     :align: center
+     :height: 480px
+     :alt: 96Boards Argonkey
+
+     96Boards Argonkey
+
+Hardware
+********
+
+96Boards Argonkey provides the following hardware components:
+
+- STM32F412CG in UFQFPN48 package
```

Comment:
STM32F412CG or ZG?

Reply:
CG

But, since it is not currently supported in Zephyr I'm using ZG, which should be same (only package differs). 
We need to add CG in Zephyr with your help after.